src/app/pages/getting-started/getting-started.html

```diff
@@ -96,16 +103,29 @@ <h2>Dependencies</h2>
                 <td>optional</td>
                 <td>No</td>
                 <td>No</td>
+                <td>No</td>
               </tr>
               <tr>
-                <td><b>11.0.0 (recommended)</b></td>
+                <td><b>11.0.0</b></td>
+                <td>16.0.0</td>
+                <td>16.0.0</td>
+                <td>No</td>
+                <td>No</td>
+                <td>optional</td>
+                <td>No</td>
+                <td>No</td>
+                <td>No</td>
+              </tr>
+              <tr>
+                <td><b>11.1.0 (recommended)</b></td>
```

Comment:
Also, I'm adding only the latest supported version for every major release.
I have to update previous rows with 
- 8.0.1
- 9.1.0
because I missed this details in previous updates.